<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crypto API GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnClassical">
        <property name="text">
         <string>Classical (RSA)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPQC">
        <property name="text">
         <string>PQC (Post-Quantum)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnKaz">
        <property name="text">
         <string>Kaz Algorithm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHybrid">
        <property name="text">
         <string>Hybrid Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="outputText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Output log will appear here...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
